events/annual-users-meeting-june-2010.html

```diff
@@ -0,0 +1,237 @@
+                <h3 class="event-title">OME and KNIME</h3>
+                <h4 class="event-speaker">Martin Horn, Konstanz</h4>
+            </div>
+        </div>
+        <div class="event-row row">
+            <div class="small-12 medium-2 columns">
+                <h5 class="event-time">12:20 - 12:40</h5>
+            </div>
+            <div class="small-12 medium-10 columns">
+                <h3 class="event-title">Fiji Is Just ImageJ – an Open Source
+                    platform for biological image analysis</h3>
+                <h4 class="event-speaker">Pavel Tomancak</h4>
+            </div>
+        </div>
+        <div class="event-row row">
+            <div class="small-12 medium-2 columns">
+                <h5 class="event-time">12:40 - 13:00</h5>
+            </div>
+            <div class="small-12 medium-10 columns">
+                <h3 class="event-title">Bio-Formats in C++ applications</h3>
+                <h4 class="event-speaker">Aaron Ponti</h4>
+            </div>
+        </div>
+        <div class="event-row row">
+            <div class="small-12 medium-2 columns">
+                <h5 class="event-time">13:00 - 14:00</h5>
+            </div>
+            <div class="small-12 medium-10 columns">
+                <h3 class="event-title">Lunch</h3>
+            </div>
+        </div>
+        <div class="event-row row">
+            <div class="small-12 medium-2 columns">
+                <h5 class="event-time">14:00 - 14:20</h5>
+            </div>
+            <div class="small-12 medium-10 columns">
+                <h3 class="event-title">Leica High Content Screening
+                    Automation and OME</h3>
+                <h4 class="event-speaker">Stefan Schek, Leica</h4>
+            </div>
+        </div>
+        <div class="event-row row">
+            <div class="small-12 medium-2 columns">
+                <h5 class="event-time">14:20 - 14:40</h5>
+            </div>
+            <div class="small-12 medium-10 columns">
+                <h3 class="event-title">Columbus – High Content solution based
+                    on OMERO</h3>
+                <h4 class="event-speaker">Karsten Kottig, PerkinElmer</h4>
+            </div>
+        </div>
+        <div class="event-row row">
+            <div class="small-12 medium-2 columns">
+                <h5 class="event-time">14:40 - 15:10</h5>
+            </div>
+            <div class="small-12 medium-10 columns">
+                <h3 class="event-title">OMERO and PSLID</h3>
+                <h4 class="event-speaker">Robert Murphy, CMU</h4>
+            </div>
+        </div>
+        <div class="event-row row">
+            <div class="small-12 medium-2 columns">
+                <h5 class="event-time">15:10 - 16:30</h5>
+            </div>
+            <div class="small-12 medium-10 columns">
+                <h3 class="event-title">Coffee &amp; Posters</h3>
+            </div>
+        </div>
+        
```

Comment:
On the original page there's one more item "16:30 - 17:30 Agenda for Day 2: Workshop Titles"